crates/sui-types/src/unit_tests/nitro_attestation_tests.rs

```diff
@@ -0,0 +1,59 @@
+    let res = verify_nitro_attestation_inner(parsed.0, parsed.1, parsed.2, now);
+    assert_eq!(
+        res.unwrap_err(),
+        SuiError::AttestationFailedToVerify(
+            "InvalidCertificate: Certificate timestamp not valid".to_string()
+        )
+    );
+}
+
```

Comment:
let's add a todo & ticket to test the parsing of different invalid attestation docs and COSE_Sign1